app/stylesheets/CameraNew.scss

```diff
@@ -0,0 +1,9 @@
+@import "./support/colors";
+.tilt-icon.tilt-icon-camera-new {
+  height: 18px;
+  width: 23px;
+  fill: white;
```

Comment:
There's actually a circle inside the camera that needs the white fill I believe.